### PR TITLE
Update docs and faq layout and styles, add .hound.yml to docs

### PR DESCRIPTION
**Pull request by frechg**
screen shot 2015-02-27 at 4 55 17 pm
screen shot 2015-02-27 at 4 55 14 pm

### CODE REVIEW

**Comment by salbertson** [comment]
@frechg, is "config" too repetitive in the Configuration nav?

**Comment by frechg** [comment]
@salbertson maybe. I just didn't want to leave it like "ruby" and nothing else. I wanted to try and be more clear about what the link will lead you to.

File: app/views/pages/configuration.haml [hunk 1]
```diff
@@ -1,137 +1,205 @@
...
+  %section.doc-content
+    %article
+      %h1 Hound Documentation
+      %p
+        Want to modify style rules to better suit your preferences?
+        Below you will find documentation on configuring style rules for each
+        language we support.
+
+      %p
+        If this documentation doesn't help you get what you where looking for.
```
Comment (salbertson) [inline app/views/pages/configuration.haml]: "If this page doesn't help you find what you are looking for, please tweet at.."?

File: app/views/pages/configuration.haml [hunk 2]
```diff
@@ -1,137 +1,205 @@
...
+
+    %article#configuration
+      %h3 Hound Configuration
+      %p
+        With a
+        %em.code .hound.yml
+        file you can tell Hound which languages not to review
+        (all supported languages are reviewed by default)
+        and specify paths to custom style rules and an ingnore file
+        for javascript.
```
Comment (salbertson) [inline app/views/pages/configuration.haml]: JavaScript?

File: app/views/pages/configuration.haml [hunk 3]
```diff
@@ -1,137 +1,205 @@
...
+      %code.code-block
+        :preserve
+          scss:
+            enabled: false
+
+          ruby:
+            config_file: .ruby-style.yml
+
+          javascript:
+            ignore_file: .javascript_ignore
```
Comment (salbertson) [inline app/views/pages/configuration.haml]: :+1:

File: app/views/pages/configuration.haml [hunk 4]
```diff
@@ -1,137 +1,205 @@
...
+
+      %p
+        In this example, review of SCSS is deactivated (:sad face:) and we are
+        pointing Hound to a custom config file for ruby style checking and
```
Comment (salbertson) [inline app/views/pages/configuration.haml]: Ruby?

File: app/views/pages/configuration.haml [hunk 5]
```diff
@@ -1,137 +1,205 @@
...
+        an ingnore file for javascript.
```
Comment (salbertson) [inline app/views/pages/configuration.haml]: JavaScript?

File: app/views/pages/faq.html.haml [hunk 6]
```diff
@@ -1,86 +1,103 @@
...
-  %p
-    For more information, see GitHub's documentation on permission levels for
-    = link_to('organization repos', 'https://help.github.com/articles/permission-levels-for-an-organization-repository/')
-    and
-    = link_to('user account repos.', 'https://help.github.com/articles/permission-levels-for-a-user-account-repository/')
+    %article
+      %h3 Why isn't Hound commenting on my pull request?
+      %ul
+        %li
+          Note that Hound only comments when he finds a style violation. Ensure
```
Comment (salbertson) [inline app/views/pages/faq.html.haml]: Remove "he" and maybe "Note that Hound only comments when style violations are found."

**Comment by srt32** [comment]
LGTM

**Comment by salbertson** [comment]
Looks great! :+1:

**Pull request closed without merging**